source/docs/example-projects/halloween-treasure-hunt.rst

```diff
@@ -0,0 +1,251 @@
+sure the event is not being triggered when clicking other types of blocks, when right clicking, or when clicking in 
+another world.
+
+Core Functionality
+==================
+
+We're ready to move further ahead and actually give a *reward* to the player clicking the block. Since we're nice 
+server owners, the prize will be a free *diamond*. This is where the :guilabel:`give` command comes in handy. Its 
+*syntax* specifies a single required argument: ``[money/xp/<item>|...]``. In our case, what we want to give the player 
+is a diamond item, so we can ignore the *money* and *xp* options.
+
+.. note::
+    When reading command documentation, It's important to keep in mind that anything inside ``< >`` is *not literal* 
+    and needs to be replaced. Arguments enclosed in ``[ ]`` are *required*, while ``( )`` means an argument is 
+    *optional*. The ``/`` symbol separates argument *options*, meaning you have to choose one of them.
+
+Let's go ahead and specify ``diamond`` as the first argument of our give command. We don't have to worry about who to 
+give the diamond to, as the command will target the linked player by default. That is, the player that triggered the 
+event. The full command line will then be ``- give diamond``.
+
+Now it's time to make sure it works. After *saving* and *reloading* scripts again, it should be giving us a *diamond* 
+every time we click the *pumpkin*. While players will totally love this, we should probably avoid giving out unlimited 
+diamonds. That's easy to fix though, we just have to *remove* the pumpkin once it's clicked. If we do it before even 
+giving out the reward, we'll make sure it won't be clicked twice. 
+
+For this, we'll use the :guilabel:`modifyblock` command, which lets us specify a *location* and a *material*. Now we 
+only need to know which location was clicked by the player. Time to make use of *context* tags! This kind of tags are 
+event specific and will let us retrieve useful information from said event. If we check again the event's 
+documentation, we can see it has a ``<context.location>`` tag available, which is just what we needed for the first 
+argument. The material, on the other hand, will be just ``air`` as we want to remove the original pumpkin. The full 
+command line will then be ``- modifyblock <context.location> air``.
+
+Our script with these new commands should look like this:
+
+.. code-block:: dscript
+    :name: figure_1_3
+    :linenos:
+    :emphasize-lines: 7,8
+
+    Halloween_Treasure_Hunt:
+    type: world
+    debug: true
+    events:
+      on player left clicks pumpkin in Hub:
+      - narrate "yay"
+      - modifyblock <context.location> air
+      - give diamond
+
+.. rst-class:: figurecaption
+
+**Figure 1.3** Our world script with core functionality
+
+Rinse and repeat: save, reload scripts and do a quick test. Amazing! This deserves a "yay". Speaking of yays… we don't 
```

Comment:
This sentence most probably can just be simplified to "Now reload your scripts and quickly test your world script" or similar. There's no need to guide the user through the process of reloading their scripts every time they need to reload their scripts. Just a quick notice is sufficient.

Reply:
Am I seeing a different line here than you @MusicScore ? The sentence you wrote is *longer* than the one you're commenting on. You didn't simplify it, you reworded it more verbosely.